### PR TITLE
minimum php version is 5.6 now

### PR DESCRIPTION
**Pull request by dbu**
* use short array notation
* use ::class pseudo-constant
* use createMock for phpunit
* simplify tests setup

### CODE REVIEW

**Comment by dbu** [comment]
@wouterj i now try to use the phpunit-bridge as well. but it looks like phpunit is not installed correctly - can you spot what i do wrong?

File: .travis.yml [hunk 1]
```diff
@@ -15,15 +12,18 @@ cache:
 
 env: SYMFONY_VERSION=3.2.*
 
+global:
```
Comment (fbourigault) [inline .travis.yml]: According to https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml, the global key must be under the env key:
```yaml
env:
    global:
        - SYMFONY_VERSION=3.2.*
        - SYMFONY_PHPUNIT_DIR=.phpunit
        - SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
        - SYMFONY_PHPUNIT_VERSION=5.6
```

Comment (dbu) [inline .travis.yml]: now i feel stupid. thanks a lot for spotting this one! that fixes it

**Comment by dbu** [comment]
ready to review and merge \o/

**Pull request merged**